app_dart/test/model/task_test.dart

```diff
@@ -55,6 +57,11 @@ void main() {
       );
       expect(Task.chromebot(commitKey: commitKey, createTimestamp: 123, builder: builder).isFlaky, isFalse);
     });
+
+    test('benchmark defaults to null', () {
+      final Task task = generateTask(1, attempts: 5);
+      expect(task.isBenchmark, null);
```

Comment:
With null safety, we should only have true or false. So this should default to false.